src/character.cpp

```diff
@@ -1445,9 +1445,31 @@ bool Character::uncanny_dodge()
     } else if( can_dodge_mut ) {
         mod_stamina( -300 );
     }
+
+    map &here = get_map();
+    const optional_vpart_position veh_part = here.veh_at( position );
+    if( in_vehicle && veh_part && veh_part.avail_part_with_feature( "SEATBELT" ) ) {
+        if( is_u ) {
+            add_msg( _( "You try to dodge, but the seatbelt is in the way!" ) );
+        } else if( seen ) {
+            add_msg( _( "%s tries to dodge, but the seatbelt is in the way!" ), this->disp_name() );
```

Comment:
Do we really need to fill the log with these messages? They are obvious, and don't need to be explicitly shown in log IMO.

Reply:
Yeah - getting rid of new strings would also allow merging it into master before 0.F.

Reply:
very well, i ll remove it